modules/system-configuration/src/main/java/org/apache/ignite/internal/configuration/SystemConfigurationSchema.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.configuration;
+
+import org.apache.ignite.configuration.annotation.ConfigurationRoot;
+import org.apache.ignite.configuration.annotation.ConfigurationType;
+import org.apache.ignite.configuration.annotation.Value;
+
+/**
+ * System Configuration schema.
+ */
+@ConfigurationRoot(rootName = "system", type = ConfigurationType.LOCAL)
+public class SystemConfigurationSchema {
+
+    /**
+     * Directory where partition data is stored. By default "partitions" subfolder of data storage path is used.
+     *
+     * <pre>
+     * The internal directory structure contains:
```

Comment:
I've heard that we wanted to be able to have raft log and partition storages on separate devices, for example. How will this configuration allow that?

Reply:
This ticket is about decoupling MS and CMG Raft logs from partitions. I'm talking about separating partitions raft logs and partitions data

Reply:
The current javadoc is misleading now. We have `partitionsBasePath` that is responsible for the log, meta and db files, while we also have `partitionsLogPath` which is also responsible for the log files? Why do we have this overlapping? Can we have three properties each responsible for its own folder?